packages/admin/cms-admin/src/pages/createEditPageNode.tsx

```diff
@@ -531,7 +531,7 @@ const transformToSlug = (name: string, locale: string) => {
 };
 
 const isValidSlug = (value: string) => {
-    return /^([a-zA-Z0-9-._~]|%[0-9a-fA-F]{2})+$/.test(value);
+    return /^[a-zA-Z0-9]+(?:[a-zA_Z0-9-_]+)*$/.test(value);
```

Comment:
According to ChatGPT, we could also do `/^[a-zA-Z0-9][a-zA-Z0-9-_]*$/, which is easier to understand